docs/design/protip.rst

```diff
@@ -0,0 +1,37 @@
+ProTip template to enhancing user experience
+============================================
+
+The ProTip template is a powerful feature that displays helpful tips to users, guiding them on how to use the platform like a pro. This template is implemented using Twig. Here's how this template works and how to effectively use it in Mautic.
+
+Template structure and functionality
```

Comment:
```suggestion
Template structure and available features
```
Maybe this works better?

Reply:
I would say that it's more about the functional part (in a sense of function, settings), than about features